tests/classification/test_dice.py

```diff
@@ -11,10 +11,60 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from functools import partial
+
 import pytest
-from torch import tensor
+import torch
+from torch import Tensor, tensor
+
+from tests.classification.inputs import _input_multiclass_prob as _input_mcls_prob
+from tests.helpers import seed_all
+from tests.helpers.testers import MetricTester
+from torchmetrics import Dice
+from torchmetrics.functional import dice, dice_score
+from torchmetrics.functional.classification.dice import _stat_scores
+
+seed_all(42)
+
+
+def _dice_score(
```

Comment:
shouldn't this be the same as 1 - dice-dissimilarity? That one is implemented at https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.distance.dice.html 

Could you verify if this is the same? Ideally we don't implement the metric twice, because then the implementation is likely the same :)